components/core/Breadcrumb.js

```diff
@@ -0,0 +1,38 @@
+const AbstractComponent = require('../AbstractComponent');
+
+/**
+ * The Breadcrumb Core Component. In addition to the Component methods, this component provides several 
+ * convience methods to set known properties. 
+ * 
+ * Based on the documentation found at:
+ * https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/breadcrumb/v2/breadcrumb
+ */
+class Breadcrumb extends AbstractComponent {
+
+    constructor() {
+        super();
+        this.props["sling:resourceType"] = "core/wcm/components/breadcrumb/v2/breadcrumb";
+    }
+
+    setStartLevel(level) {
+        this.props["startLevel"] = level;
+    }
+
+    setShowHidden(hidden) {
```

Comment:
For this and next three methods: `setShowHidden, setHideCurrent, setDisableShadowing` the argument name is ambiguous / implies the opposite. Example: `setShowHidden(hidden)` would indicate to the caller that the argument `hidden` is true/false, but providing `true` would actually result in `showHidden=true`, which is the opposite intent. 
Easiest way to communicate is to simply repeat the variable after the _set_ word: `setShowHidden(showHidden), setHideCurrent(hideCurrent)...`